app/Listeners/PersistTweetId.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Listeners;
+
+use App\Notifications\PostArticleToTwitter;
+use Illuminate\Notifications\Events\NotificationSent;
+
+final class PersistTweetId
```

Comment:
Bit of a personal taste but I feel like the below name sounds a tad better. Feel free to ignore it if you don't agree:

```suggestion
final class StoreTweetIdentifier
```